### PR TITLE
created two different forms

### PR DESCRIPTION
**Pull request by Banjer71**
I created two different forms, one as a class component and one as a functional component

### CODE REVIEW

File: src/components/Form/Form2.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const Form2 = () => {
```
Comment (shayanypn) [inline src/components/Form/Form2.js:3]: @Banjer71 Why there is any need to create a new component? even if there is any need reason, why name it this way

File: src/components/Form/index.jsx [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+    const btn = document.getElementById("btn");
+
+    const handleUpload = () => {
+      btn.addEventListener("click", () => {
+        upload.click();
+      });
+    };
+    handleUpload();
+  }
+
```
Comment (shayanypn) [inline src/components/Form/index.jsx:16]: @Banjer71 you can create a method and directly call it via `onClick` event.

File: src/components/Form/Form2.js (hunk 1 above)
Comment (Banjer71) [reply src/components/Form/Form2.js:3]: I removed it

File: src/components/Form/index.jsx [hunk 3]
```diff
@@ -1,13 +1,20 @@
 import React, { Component } from "react";
 
 class Form extends Component {
+  handleUpload = () => {
+    upload.click();
```
Comment (shayanypn) [inline src/components/Form/index.jsx:5]: @Banjer71 it's better to make reference for this element, using `ref` prop of element.

Comment (Banjer71) [reply src/components/Form/index.jsx]: I added the ref

**Pull request merged**